app/common/templates/common/macros/collections.html

```diff
@@ -134,69 +134,161 @@ <h1 class="govuk-heading-l">{{ runner.question_page_heading }}</h1>
+  {% for question in submission.cached_get_ordered_visible_questions(container, override_context=context if add_another_index is not none else none) %}
+    {# fixme: set if we're in an add another context at the top of this macro and use that everywhere we do add_another_index is not none #}
+    {% if question.add_another_container and add_another_index is none %}
+      {# for the add another context we'll just include one row with a link to the full answers below #}
+      {% if question.add_another_container not in (add_another_contexts | map(attribute="container")) %}
+        {% set count = submission.get_count_for_add_another(question.add_another_container) %}
+        {% set summaries = [] %}
+        {% set status = namespace(all_answered = true) %}
+        {% for i in range(count) %}
+          {% set (summary, is_answered) = submission.get_answer_summary_for_add_another(question.add_another_container, add_another_index=i) %}
+          {%
+            do summaries.append({
+              "summary_text_line": summary,
+              "is_answered": is_answered
+            })
+          %}
+          {% if not is_answered %}{% set status.all_answered = false %}{% endif %}
+        {% endfor %}
+        {%
+          do add_another_contexts.append({
+            "container": question.add_another_container,
+            "count": count,
+            "summaries": summaries,
+            "all_answered": status.all_answered
+          })
+        %}
 
-      {# todo: can we "anchor" or link to the specific question if we're going to be changing a question on a same page group #}
-      {%
-        set actions = (
-        []
-        if submission.is_completed else
-        [
-          {
-            "href": runner.to_url(enum.form_runner_state.QUESTION, question=question, source=enum.form_runner_state.CHECK_YOUR_ANSWERS),
-            "text": "Change",
-            "visuallyHiddenText": question.name
-          }
-        ]
-        )
-      %}
-      {%
-        do rows.append({
-          "key": {"text": runner.interpolate(question.text)},
-          "value": {"html": value_html},
-          "actions": {"items": actions},
-        })
-      %}
+        {% set key_text = runner.interpolate(question.add_another_container.name) %}
+
+        {# the add another context has been fully answered and should be checked on this page #}
+        {% if count and status.all_answered %}
+          {% set value_html %}
+            <p class="govuk-body">You have added {% trans count=count %}{{ count }} answer{% pluralize %}{{ count }} answers{% endtrans %}</p>
+
+            <p class="govuk-body">
+              <a href="#{{ question.add_another_container.id }}-answers" class="govuk-link govuk-link--no-visited-state">Answers for {{ question.add_another_container.name | lower }}</a>
```

Comment:
Using `lower` on user text like this always feels risky - will downcast any proper nouns or similar, which isn't a safe thing for us to do 🤔

Reply:
Quick alternative option: stick it in quotes and keep the capitalisation? 😬